containers/proxy-systemd-services/uyuni-proxy-systemd-services.spec

```diff
@@ -22,8 +22,8 @@ Name:           uyuni-proxy-systemd-services
 Summary:        Uyuni proxy server systemd services containers
 License:        GPL-2.0-only
 Group:          Applications/Internet
-Version:        5.0.1
-Release:        1
+Version:        5.1.0
```

Comment:
@deneb-alpha I think we discuss this package an it can be removed, right? This is the systems package for proxy that was replaced by the mgrpxy.

Reply:
Yes, having it around creates needless confusion and it would be better if we can remove it. Anyways, not as part of this PR and can be done separately afterwards.

Reply:
Fyi @rjmateus this is addressed in the PR description.

Reply:
thank you @Etheryte I should have started by reading the description :)